pep_sphinx_extensions/pep_theme/static/style.css

```
@@ -33,7 +33,8 @@
 /* Set master colours */
 :root {
     --colour-background: var(--light, white) var(--dark, #011);
-    --colour-background-accent: var(--light, #ccc) var(--dark, #333);
+    --colour-background-accent-strong: var(--light, #ccc) var(--dark, #333);
+    --colour-background-accent-light: var(--light, #ddd) var(--dark, #222);
```

```suggestion
    --colour-background-accent-light: var(--light, #ddd) var(--dark, #111);
```

Mute the colors as much as practical (since we're adding back the lines to clearly delineate the rows)